src/dao/RegistrationDAO.js

```diff
@@ -345,6 +348,24 @@ async function getComponentInfo (challengeId) {
   return helper.queryDataFromDB(QUERY_GET_COMPONENT_INFO, [challengeId, challengeId, challengeId, challengeId, challengeId])
 }
 
+const QUERY_UPDATE_COMPONENT_VERSION_INFO = `
+UPDATE comp_versions
+SET rating = ?
+WHERE comp_vers_id = ?
+`
+
+/**
+ * Update component version info
+ * @param componentVersionId the componentVersionId to pass
+ * @param rating the rating to pass
+ * @param componentVersionId
+ * @param rating
+ * @return the result
+ */
+async function updateComponentVersionInfo (componentVersionId, rating) {
+  return helper.executeSQLonDB(QUERY_UPDATE_COMPONENT_VERSION_INFO, [rating, componentVersionId])
+}
+
```

Comment:
there is no "rating" column in comp_versions table

Reply:
Hmm.. where do we save ratings then? Can you please give me the SQL for that?

Reply:
It is not required